lib/src/services/coin_service/app_coin_service.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+import '/../src/config/app_config.dart';
+import '/../src/config/constants.dart';
+
+import '/../src/models/coins.dart';
+import '/../src/services/coin_service/coin_service.dart';
+
+class AppAlbumService extends CoinService {
+  AppAlbumService(http.Client client) : super(client);
+
+  @override
+  Future<List<Coins>> fecthCoins() async {
+    Map<String, String> qParams = {
```

Comment:
qParams what does it mean?

Reply:
That's queryParameters. I will change the name.